### PR TITLE
README.md: deprecate and move to Homebrew

### PR DESCRIPTION
**Pull request by iMichka**

### CODE REVIEW

**Review by MikeMcQuaid — APPROVED** [review]

**Comment by sjackman** [comment]
> User repositories have been automatically migrated to Homebrew since brew 2.0.0.

This part's not yet true. We need to write that code for `brew update` to automatically migrate people to Homebrew/brew.

I'd like this PR to be 🍏. Can we revert the changes that break `brew man --fail-if-changed` or alternatively fix `brew man --fail-if-changed` itself?

File: Library/Homebrew/dev-cmd/man.rb [hunk 1]
```diff
@@ -55,6 +55,10 @@ def man_args
   end
 
   def man
+    # Disabled to make CI pass, as we removed most of the content of the REAME.md file,
+    # and build_man_page would fail.
+    return if ENV["CI"]
```
Comment (MikeMcQuaid) [inline Library/Homebrew/dev-cmd/man.rb:60]: I'd suggest just removing CI entirely instead and/or adjusting the `azure-pipelines.yml` file to be a no-op.

**Review by MikeMcQuaid — COMMENTED** [review]
Suggestion: close all issues/PRs and archive this repository when it is merged. Could even be moved to the Homebrew organisation after archive so we can make changes in future if needed and control access more easily (we did similar for Caskroom).

File: Library/Homebrew/dev-cmd/man.rb (hunk 1 above)
Comment (iMichka) [reply Library/Homebrew/dev-cmd/man.rb:60]: Yes, that's probably also a good idea. I'll update my PR accordingly.

**Comment by iMichka** [comment]
@sjackman @MikeMcQuaid We need to write the migration script for our our previous users. Once this is merged, I will update this PR and we will be able to tag 1.9.4. Once this is done we will archive this repo.

**Comment by sjackman** [comment]
I'm in favour of archiving this repo after the migration PR is merged.
I'm fine with moving it to the Homebrew organization. What name though? `Homebrew/linuxbrew`? That name currently redirects from there to https://github.com/Linuxbrew/legacy-linuxbrew. We could kill that redirect by moving this repo.

I'd rather not disable CI. We may need it if for some unreason we need to fix a bug (for example in the migration code).

**Pull request closed without merging**